src/test/java/net/mcreator/integration/TestSetup.java

```diff
@@ -58,16 +59,19 @@ public static void setupIntegrationTestEnvironment() throws IOException {
 		conf.load(Launcher.class.getResourceAsStream("/mcreator.conf"));
 		Launcher.version = new MCreatorVersionNumber(conf);
 
-		// init theme
+		// load plugins
+		// We begin by loading plugins, so every image can be changed
+		PluginLoader.initInstance();
+		// We load UI themes now so we can check if we can still use it
+		ThemeLoader.initUIThemes();

```

Comment:
"so we can check if we can still use it" What did you mean here? :D Also add one space before this comment for code clarity.

Reply:
I will change it, but what I think I meant is "so we can check if we can still use the theme defined by the user" or something similar to this.

Reply:
Maybe say: we load UI themes now as theme plugins are loaded at this point